ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/objectstore/StoreManager.java

```diff
@@ -145,20 +145,20 @@ private static final ObjectStoreAPI initStore(String name)
     {
         ObjectStoreEnvironmentBean storeEnvBean = BeanPopulator.getNamedInstance(ObjectStoreEnvironmentBean.class, name);
         String storeType = storeEnvBean.getObjectStoreType();
-        ObjectStoreAPI store;
 
         try
         {
-            store = ClassloadingUtility.loadAndInstantiateClass(ObjectStoreAPI.class, storeType, name);
+            ObjectStoreAPI store = ClassloadingUtility.loadAndInstantiateClass(ObjectStoreAPI.class, storeType, name);
+
+            store.start();
```

Comment:
So the PR has not introduced any behavioural change. I therefore reject it unless you can convince me that the change adds benefit.